tools/gen_javadoc.py

```diff
@@ -63,9 +65,17 @@ def main():
     '-d', args.out_dir,
     '-link', 'https://developer.android.com/reference/',
   ] + packages
-  print(' '.join(command))
 
-  return subprocess.call(command)
+  if not args.quiet:
+    print(' '.join(command))
+
+  try:
+    subprocess.check_output(command, stderr=subprocess.STDOUT)
+    if not args.quiet:
+      print(output)
+  except subprocess.CalledProcessError as e:
+    print(e.output)
+    return e.returncode
```

Comment:
I changed this since you reviewed @blasten 

This will supress the output from the process unless there's an error, or the command was invoked without the `--quiet` flag.